integration_tests/pyproject.toml

```diff
@@ -1,18 +1,17 @@
 [tool.poetry]
-name = "tasman-dbt-revenuecat"
+name = "tasman-dbt-utils"
 version = "0.0.1"
-description = "Package to transform RevenueCat data into actionable datasets."
+description = "Package with useful utility features."
 authors = ["Tasman"]
-readme = "README.md"
+readme = "../README.md"
 
 [tool.poetry.dependencies]
-python = "^3.9"
+python = "^3.12"
```

Comment:
Does this need changing back to a version that is supported?

Reply:
No I forgot to update the python version in the CI, done that now and CI passes :)